README.md

```diff
@@ -1,50 +1,22 @@
-# jekyll-bootstrap-theme
+# How Does This Work?
 
-Jekyll theme w/ Bootstrap; see a live demo @ [`henrythemes.github.io/jekyll-bootstrap-theme` »](http://henrythemes.github.io/jekyll-bootstrap-theme)
+We use a Jekyll custom theme for markup and display, and pages are published to http://projecthydra.github.io.
 
+## Adding or Editing Content
 
-Note: The theme uses the bootstrap sass/scss (source) version
-letting you change (override) all bootstrap css variables.
+These are community documents, so we rely on the pull request model. If you'd like to contribute content:
 
-Example:
+- clone the project
+- make a branch  
+- create/edit pages within the Hydra directory
+- add their links to home_sidebar.yml
+- tag them appropriately
 
-~~~
-tbd
-~~~
 
+## jekyll-bootstrap-theme
 
-Note: GitHub Pages has built-in support for sass/scss, thus, your static
-site will result in a single all-in-one-file stylesheet, that is, `style.css`.
+Jekyll theme w/Bootstrap [`henrythemes.github.io/jekyll-bootstrap-theme` »](http://henrythemes.github.io/jekyll-bootstrap-theme)
```

Comment:
This isn't true anymore, though, right? Our new version uses the Documentation theme (http://idratherbewriting.com/documentation-theme-jekyll/), and more specifically, a gemified version of it that @elrayle wrote.